packages/styling/src/styles/scheme.ts

```diff
@@ -0,0 +1,35 @@
+import { mergeCustomizations, ICustomizerContext } from '@uifabric/utilities';
+import { IScheme, ITheme } from '../interfaces/index';
+
+/**
+ * Modify context to activate the specified scheme, using scheme in context (if available)
+ * and falling back to settingsTheme. If no scheme is available, undefined context will be returned.
+ *
+ * @param scheme Scheme to get customizations for.
+ * @param context Context in which to get schemed customizations.
+ * @param settingsTheme Themes setting fallback if context does not have schemed theme.
+ */
+export function getSchemedContext(scheme: IScheme, context: ICustomizerContext, settingsTheme?: ITheme): ICustomizerContext | undefined {
+  // As written now, schemes affect all components (including styled components) by modifying the active contextual theme.
```

Comment:
Thanks for the detailed comments!

I think we may need a "themeThatIsForThisContext" and "themeToActuallyUseInStyles". They are usually the same, but sometimes the latter is a scheme of the former.

Reply:
I'm not sure what you mean by "need". The `scheme` component prop, if present, affects the active contextual theme. If `theme` component prop is also provided, it will override the contextual theme here, but only for the localized component being created. The contextual theme will be passed on down the hierarchy either way. Maybe I'm missing something, but I don't see why whether one is a scheme of the other matters...

Reply:
In other words, the `theme` component prop doesn't even play into this function.. this function only takes in inputs that can affect the context, and `theme` component prop does not play here.